ruff.toml

```diff
@@ -14,12 +14,16 @@ extend-select = [
 	"ANN2", # missing-return-type-*
 	"FA", # flake8-future-annotations
 	"F404", # late-future-import
+	"PT", # flake8-pytest-style
 	"PYI", # flake8-pyi
 	"UP", # pyupgrade
 	"TRY",
 	"YTT", # flake8-2020
 ]
 ignore = [
+	"PT007", # temporarily disabled, TODO: configure and standardize to preference
+	"PT011", # temporarily disabled, TODO: tighten expected error 
```

Comment:
I think I'd like to see https://github.com/astral-sh/ruff/issues/5157 or https://github.com/astral-sh/ruff/issues/6840 before enabling `PT011`